services/app-web/src/pages/StateSubmission/RateDetails/SingleRateCert/SingleRateCert.tsx

```diff
@@ -0,0 +1,548 @@
+                                <span className="padding-top-1">
+                                    {`Upload any supporting documents for Rate certification ${rateCertNumber}`}
+                                </span>
+                                <span>
+                                    {supportingDocsByRate
+                                        ? 'Additional rates can be added later.'
+                                        : 'Additional rates and supporting documents can be added later.'}
+                                </span>
+
+                                <span className="padding-top-1">
+                                    This input only accepts PDF, CSV, DOC, DOCX,
+                                    XLS, XLSX, XLSM files.
+                                </span>
+                            </span>
+                        }
+                        accept={ACCEPTED_SUBMISSION_FILE_TYPES}
+                        initialItems={rateInfo.supportingDocuments}
+                        uploadFile={(file) =>
+                            handleUploadFile(file, 'HEALTH_PLAN_DOCS')
+                        }
+                        scanFile={(key) =>
+                            handleScanFile(key, 'HEALTH_PLAN_DOCS')
+                        }
+                        deleteFile={(key) =>
+                            handleDeleteFile(
+                                key,
+                                'HEALTH_PLAN_DOCS',
+                                previousDocuments
+                            )
+                        }
+                        innerInputRef={multiRatesConfig?.reassignNewRateRef}
+                        onFileItemsUpdate={({ fileItems }) =>
+                            setFieldValue(
+                                `${fieldNamePrefix}.supportingDocuments`,
+                                fileItems
+                            )
+                        }
+                    />
+                </FormGroup>
+            )}
+
+            {showPackagesWithSharedRatesDropdown && (
+                <PackagesWithSharedRates
+                    index={index}
+                    key={key}
+                    fieldNamePrefix={fieldNamePrefix}
+                    shouldValidate={shouldValidate}
+                    parentSubmissionID={parentSubmissionID}
+                />
+            )}
+
+            <FormGroup error={Boolean(showFieldErrors('rateProgramIDs'))}>
+                <Label htmlFor={`${fieldNamePrefix}.rateProgramIDs`}>
+                    Programs this rate certification covers
+                </Label>
+                <PoliteErrorMessage>
+                    {showFieldErrors('rateProgramIDs')}
+                </PoliteErrorMessage>
+                <ProgramSelect
+                    name={`${fieldNamePrefix}.rateProgramIDs`}
+                    inputId={`${fieldNamePrefix}.rateProgramIDs`}
+                    programIDs={rateInfo.rateProgramIDs}
```

Comment:
I noticed an error in the console when clicking on the program select validation error in the error summary.

Reply:
Oops! here is the rest of the info for this error
![Screenshot 2023-06-27 at 10 16 59 AM](https://github.com/Enterprise-CMCS/managed-care-review/assets/98117700/69854a11-8f35-403e-a04f-e133f437bdb7)

This error is from `ErrorSummary` component using `querySelector` to focus on the combo box. `querySelector` does not like special characters when passed in in the format `#rateInfos.0.rateProgramIDs`.

To fix this we just need to change the `fieldSelector` value to `[id=$:{errorKey}"]`
![Screenshot 2023-06-27 at 10 21 12 AM](https://github.com/Enterprise-CMCS/managed-care-review/assets/98117700/8ed566dd-bfc1-449d-a09b-208a9d183539)

Reply:
@rswerve I think I might have misled you by highlighting the else block. The code that needs to be changed in on line 18. From
```typescript
 fieldSelector = errorKey
```
to 
```typescript
fieldSelector = `[id="${errorKey}"]`
```